<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDialog</class>
 <widget class="QDialog" name="CameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机自动聚焦测试程序</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>曝光时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ExpuseTime">
         <property name="text">
          <string>150000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>增益</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gain">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenCamera">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>打开相机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCloseCamera">
         <property name="text">
          <string>关闭相机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAbort">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>电机</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labelMotorCom">
            <property name="text">
             <string>电机端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="MotorCom">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>最小移动步长</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="MinStep">
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>最大移动步长</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="MaxStep">
            <property name="text">
             <string>3000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>聚焦误差系数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Epsylon">
            <property name="text">
             <string>0.05</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnectMotor">
            <property name="text">
             <string>连接电机</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCloseMotor">
            <property name="text">
             <string>关闭电机</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBigFocus">
         <property name="text">
          <string>粗调</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAutoFocus">
         <property name="text">
          <string>聚焦（微调）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAbortFocus">
         <property name="text">
          <string>停止聚焦</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="zoomOutIcon">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/zoomout.png</normaloff>:/zoomout.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="zoomSlider">
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>250</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoomInIcon">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/zoomin.png</normaloff>:/zoomin.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>相机温度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTemper">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>聚焦时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_focusTime">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>脉冲量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pulses"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnX">
           <property name="text">
            <string>+X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNX">
           <property name="text">
            <string>-X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btnY">
           <property name="text">
            <string>+Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNY">
           <property name="text">
            <string>-Y</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnZ">
           <property name="text">
            <string>+Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNZ">
           <property name="text">
            <string>-Z</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
